packages/sinuous/src/index.d.ts

```diff
@@ -101,8 +101,12 @@ declare namespace sinuous {
    * Sinuous internal API.
    */
   interface Api extends Options {
-    insert<T>(el: Node, value: T, marker?: Node, current?: T, startNode?: Node): T;
+    h: typeof h;
+    hs: typeof hs;
+    insert<T>(el: Node, value: T, endMark?: Node, current?: T, startNode?: Node): T;
```

Comment:
I put these in the order of _sinuous/packages/sinuous/h/src/index.js_. Felt right to include h and hs, and you hadn't updated the parameter of `insert`'s `marker` to be `endMark`